docs/raph-rover/guides/software-update.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Software update
+sidebar_label: Software update
+sidebar_position: 7
+description: >-
+  Detailed guide on updating the software of Raph Rover, covering steps to
+  download and flash the latest RaphOS image.
+---
+
+## Flashing a new RaphOS image
+
+In order to update the Raph Rover software, you will need a USB drive with at
+latest 8GB of storage and software for flashing images like
+[Balena Etcher](https://etcher.balena.io/).
+
+### Download the newest RaphOS image
+
+You can find all releases of RaphOS
+[here](https://github.com/RaphRover/RaphOS/releases). Choose the newest one and
+download the image (.iso file).
+
+### Flash image to USB drive
+
+#### Using Etcher
+
+Open [Etcher](https://www.balena.io/etcher/) and point it to your new USB drive
+image location (.iso file) via `Flash from file`.
+
+Click `Select target` option and choose your USB drive, then click `Flash!`.
+
+<ImageZoom
+  src="/img/robots/raph/guides/software-update/etcher.webp"
+  alt="Balena Etcher window"
+  width="795"
+  height="503"
+/>
+
+After the flashing completes, disconnect the USB drive from your computer.
+
+### Install the new image on Raph Rover
+
+Make sure that:
+
+- The rover is powered off
+- The rover is not plugged in to a charger
+- The rover's batteries have at least 50% charge.
+
+:::tip
+
+To see the progress of the installation, you can connect a monitor to the
+rover's HDMI port. Make sure to do that before powering on the rover.
+
+:::
+
+Insert the USB drive with the new image into the USB port on the back of the
+rover and power on the rover.
+
+:::info
+
+The rover will boot from the USB drive and start the installation process
+automatically after powering on.
+
+Installation process takes about 5 minutes.
+
+:::
+
+After 5 minutes have passed, the new image should be installed. You can now
+remove the USB drive and power cycle the rover by holding the power button to
+turn it off and then turning it back on.
```

Comment:
The installation script will automatically reboot the on-board computer after removing the USB drive. There's no need to power cycle the whole robot.